js/wee.routes.js

```diff
@@ -127,9 +130,13 @@
 					if (eq) {
 						// If ran is true then execute the route immediately
 						if (ran) {
-							W.$exec(child, {
-								args: seg
-							});
+							if (object) {
+								_process(child, y, total, event, [opt, seg]);
```

Comment:
It is just a result of the route nesting functionality since an object would require child property processing.